spec/cog_cmd/ecs/container/list_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe 'the container-list command' do
+  let(:command_name) { 'container-list' }
+
+  let(:client) do
+    client = Object.new
+    allow(Aws::S3::Client).to receive(:new).and_return(client)
+    client
+  end
+
+  let(:object_list) do
+    obj = Aws::S3::Types::Object.new(key: 'container-definitions/mydef.json',
+                                     last_modified: '2016-09-06 23:39:15 UTC')
+    Aws::S3::Types::ListObjectsV2Output.new(contents: Array.new.push(obj))
```

Comment:
I think you can use an array literal here: `Aws::S3::Types::ListObjectsV2Output.new(contents: [obj])`